packages/dev/core/src/Meshes/geometry.ts

```diff
@@ -78,6 +78,7 @@ export class Geometry implements IGetSetVerticesData {
     public _softwareSkinningFrameId: number;
     private _vertexArrayObjects: { [key: string]: WebGLVertexArrayObject };
     private _updatable: boolean;
+    private _hasVertexAlpha = false;

```

Comment:
just make hasVertexAlpha public, no need for a property ?

Reply:
good point, will change